ui/src/me/selectors/index.ts

```diff
@@ -0,0 +1,20 @@
+// Libraries
+import {get} from 'lodash'
+
+// Types
+import {AppState} from 'src/types'
+import {MeState} from 'src/shared/reducers/me'
+
+// TODO(desa): is this reasonable or can I trust app state?
+const emptyMeState: MeState = {
```

Comment:
`AppState` is initialized in the [root node of the application] by the redux `Provider` (https://github.com/influxdata/influxdb/blob/master/ui/src/index.tsx#L74). The reducer which provides the `me` initial state is [located here](https://github.com/influxdata/influxdb/blob/master/ui/src/shared/reducers/me.ts#L14
).  We are guaranteed that we will (at a minimum) have this empty me state.  

So this `emptyMeState` is redundant and can be removed.